src/mlpack/core/cv/metrics/topk_accuracy.hpp

```diff
@@ -0,0 +1,59 @@
+/**
+* @file topk_accuracy.hpp
+* @author Arunav Shandeelya
+*
+* The TopK Accuracy [target is in topk prediction] Classification metric
+
+
+* mlpack is free software; you may redistribute it and/or modify it under the
+* terms of the 3-clause BSD license.  You should have received a copy of the
+* 3-clause BSD license along with mlpack.  If not, see
+* http://www.opensource.org/licenses/BSD-3-Clause for more information.
+*/
+
+#ifndef MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_HPP
+#define MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_HPP
+
+#include <mlpack/core.hpp>
+
+namespace mlpack {
+namespace cv{
+
+/**
+ * The topk accuracy is a metric for classification algorithms
+ * which gives top 'K' highest probability of the model. This metric
+ * computes the number of times where the correct label is among the 
+ * top 'K' label predicted (rank by predicted score).   
+*/
+
+class TopK_Accuracy
+{
+    public:
+    /**
+     * Run Prediction and calculate the topk accuracy score.
+     * 
+     * @param model A classification model
+     * @param data Column-major containing test-items.
+     * @params labels (Ground truth), target values for the test_items 
+     * @params K, the top k label predicted class.
```

Comment:
I think this should be small (line 44). Let me know what you think.

Reply:
Yeah, it's looking good now. added